src/box/lua/config/applier/box_cfg.lua

```diff
@@ -789,23 +907,27 @@ local function apply(config)
 
         -- RO may be enforced by the isolated mode, so we call the
         -- function after all the other RO/RW logic is done.
-        switch_isolated_mode(configdata, box_cfg)
+        switch_isolated_mode_before_box_cfg(configdata, box_cfg)
 
         log.debug('box_cfg.apply (startup)')
         box.cfg(box_cfg)
 
+        switch_isolated_mode_after_box_cfg(config)
```

Comment:
Needs some update after [changes](https://github.com/tarantool/tarantool/pull/10811#discussion_r1850563921) in the previous patch.

Reply:
Updated.